src/fswAlgorithms/orbitControl/lambertSurfaceRelativeVelocity/lambertSurfaceRelativeVelocity.rst

```diff
@@ -0,0 +1,69 @@
+Executive Summary
+-----------------
+This module computes the required inertial spacecraft velocity :math:`{}^N\mathbf{v}_{B/N}` to satisfy the desired
+relative velocity to the surface :math:`{}^S\mathbf{v}_{rel,des}` at position :math:`{}^N\mathbf{r}_{B/N}` and
+(maneuver) time :math:`t`. The module takes into account the rotation and orientation of the celestial body, provided
+by the :ref:`EphemerisMsgPayload`. The spacecraft position :math:`{}^N\mathbf{r}_{B/N}` is provided by the second
+position vector :math:`{}^N\mathbf{r}_{2} in `:ref:`LambertProblemMsgPayload`, as this module is designed to be used at
```

Comment:
RST error here.  remove the symbol before `:ref:`

Reply:
Fixed it.